tests/test_helpers.py

```diff
@@ -84,10 +73,20 @@
      "4. planning meeting    due date: 2020-05-30    converged\n"
      "5. deliverable    due date: 2021-05-05    finalized\n"
      ),
+    (["helper", "u_logurl", "20sb", "-n 1", "this is a test"],
```

Comment:
I think this should be 
```
"helper", "u_logurl", "20sb", "-n", "1", "https://docs.google.com/document/d/1pQMFpuI"
```
(made a more realistic url and split the -n command and its value)

Reply:
Got it! Also please don't mind the fact that I accidentally hit the close button thinking it was going to close the comment and instead closed the pull request haha oops